packages/react-sdk/src/tree/webstudio-component.tsx

```diff
@@ -40,13 +40,16 @@ export const WebstudioComponent = ({
   components,
   ...rest
 }: WebstudioComponentProps) => {
-  const instanceProps = useInstanceProps(instance.id);
+  const { __show = true, ...instanceProps } = useInstanceProps(instance.id);
   const props = {
     ...instanceProps,
     ...rest,
     [idAttribute]: instance.id,
     [componentAttribute]: instance.component,
   };
+  if (__show === false) {
```

Comment:
not sure ... maybe the property name could come from a constant that is part of that api? so that what's inside doesn't matter? Value could be even a symbol?

Reply:
Or could be just data-ws-show maybe in align with id and component props?